### PR TITLE
Add Github Actions workflow files

### PR DESCRIPTION
**Pull request by jimmidier**
Related to #82

### CODE REVIEW

File: .github/workflows/SSW.SophieBot.yml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    paths:
+      - src/SSW.SophieBot/**
+      - .github/workflows/SSW.SophieBot.yml
+      - .github/workflows/Build-LUIS.ps1
+      - .github/workflows/Build-Orchestrator.ps1
+      - .github/workflows/LUUtils.ps1
+      - .github/workflows/PublishUtils.ps1
+      - .github/workflows/Publish-WebApp.ps1
+
+concurrency:
```
Comment (PatrickZhao1989) [inline .github/workflows/SSW.SophieBot.yml:16]: Why do we need this?

File: .github/workflows/SSW.SophieBot.yml [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+          dotnet-version: '3.1.x'
+      
+      - name: Create appsettings file
+        uses: jsdaniell/create-json@1.1.2
+        with:
+          name: "appsettings.json"
+          json: ${{ secrets.APPSETTINGS_JSON }}
+          dir: "src/SSW.SophieBot/SSWSophieBot/settings"
+      
+      - name: Set environment variables
```
Comment (PatrickZhao1989) [inline .github/workflows/SSW.SophieBot.yml:52]: You'd better explain to me what this section does

**Review by PatrickZhao1989 — COMMENTED** [review]
Refer to the comment

File: .github/workflows/SSW.SophieBot.yml (hunk 1 above)
Comment (jimmidier) [reply .github/workflows/SSW.SophieBot.yml:16]: These two ps scripts are for publishing to Azure app service. They mainly do appsettings.json merging.

File: .github/workflows/SSW.SophieBot.yml (hunk 2 above)
Comment (jimmidier) [reply .github/workflows/SSW.SophieBot.yml:52]: "Create appsettings file" section is for creating appsettings.json from GitHub secret,
"Set environment variables" section is for extracting Azure keys and credentials from the publishing profile in appsettings.json into environment variables, so they can be used by later steps

File: .github/workflows/SSW.SophieBot.yml (hunk 1 above)
Comment (PatrickZhao1989) [reply .github/workflows/SSW.SophieBot.yml:16]: Sorry I was asking why we need concurrency setting

File: .github/workflows/SSW.SophieBot.yml (hunk 2 above)
Comment (PatrickZhao1989) [reply .github/workflows/SSW.SophieBot.yml:52]: Can't later steps have environment variables respectively?

File: .github/workflows/SSW.SophieBot.yml (hunk 1 above)
Comment (jimmidier) [reply .github/workflows/SSW.SophieBot.yml]: Concurrency setting is for ensuring only one workflow per env will run at a time

File: .github/workflows/SSW.SophieBot.yml (hunk 2 above)
Comment (jimmidier) [reply .github/workflows/SSW.SophieBot.yml]: I have done refinements for env variables settings and usages. See latest commits.

**Review by PatrickZhao1989 — APPROVED** [review]

**Pull request merged**